components/automate-ui/src/app/pages/+compliance/+reporting/reporting.component.ts

```diff
@@ -445,7 +446,7 @@ export class ReportingComponent implements OnInit, OnDestroy {
     const foundFilter = urlFilters.find( (filter: Chicklet) => filter.type === 'end_time');
 
     if (foundFilter !== undefined) {
-      const endDate = moment(foundFilter.text, 'YYYY-MM-DD');
+      const endDate = moment(foundFilter.text + 'GMT+00:00', 'YYYY-MM-DDZ');
```

Comment:
parsing the date as UTC. Before the date would come in as the local users timezone.